Create_Parallels_DaaS_Prereqs_Azure/Create_Parallels_DaaS_Prereqs_Azure.ps1

```diff
@@ -498,14 +499,9 @@ function add-AzureAppRegistrationPermissions {
 
     # Define the desired permissions
     $desiredApplicationPermissions = @(
-    "Group.ReadWrite.All",
-    "GroupMember.ReadWrite.All",
-    "User.Read.All",
-    "User.Export.All",
-    "User.Invite.All",
-    "User.ManageIdentities.All",
-    "User.ReadWrite.All",
-    "Domain.Read.All"
```

Comment:
I think this one is needed to read Custom Domains. @DenisFuryaev isn't it?

Reply:
@esmirnov this change was validated by QA in PVAD-5405.

Reply:
Yes, `Domain.Read.All` permission is needed, but I see that it is still present on line 503, so it was not removed from the file